src/rules/no-import-module-exports.js

```diff
@@ -0,0 +1,72 @@
+import minimatch from 'minimatch'
+import path from 'path'
+import readPkgUp from 'read-pkg-up'
+
+function getEntryPoint(context) {
+  try {
```

Comment:
If you require.resolve a dir path that contains a package.json, it gives you the path to the "main"